spec/collectionspace/mapper/data_mapper_core_spec.rb

```diff
@@ -162,8 +162,9 @@
           expect(diff).to eq([]) 
         end
 
-        it 'maps as expected' do puts
+        it 'maps as expected' do
           @fixture_xpaths.each do |xpath|
+            next if xpath.start_with?('/document/objectexit_common/exitDateGroup')
```

Comment:
If you run into an xpath that is failing on this one, add something to the code (see instruction page for examples) to skip it so that the rest of the xpaths will be checked. 

Here, all the other xpaths pass the tests.